src/background.ts

```diff
@@ -107,6 +107,20 @@ app.on('ready', async () => {
   createFirstWindow()
 })
 
+// Open a connection from a file (e.g. ./sqlite.db)
+app.on('open-file', (event, file) => {
+  event.preventDefault();
+  const win = getActiveWindows()[0];
```

Comment:
This should probably be creating a new window?. I'm also not handing this:

> open-file is also emitted when a file is dropped onto the dock and the application is not yet running. Make sure to listen for the open-file event very early in your application startup to handle this case

Reply:
Hey! Yes I think it should definitely open a new window. I can do that.